packages/next/src/server/route-modules/app-route/module.ts

```diff
@@ -350,6 +350,7 @@ export class AppRouteRouteModule extends RouteModule<
           const prospectiveRoutePrerenderStore: PrerenderStore =
             (prerenderStore = {
               type: 'prerender',
+              phase: 'render',
```

Comment:
@sebmarkbage i'm not sure how to handle the case of prerendering route modules -- giving them `phase: 'action'` feels a bit strange, because they definitely shouldn't be able to do anything mutable. but i guess we'd block that by other means (i.e. dynamic APIs throwing during a prerender) so maybe it's fine?